web/styles/app_variables.css

```diff
@@ -55,6 +55,8 @@
     */
     --left-sidebar-collapse-widget-gutter: 10px;
     --left-sidebar-width: 270px;
+    /* 15px (left margin) + 40px (toggle icon) + 10px (margin) + 20px logo + 10px right margin */
+    --left-sidebar-width-with-realm-icon-logo: 95px;
```

Comment:
I don't love how fragile that is. Probably long-term, we should express this as a `calc` adding together those other variables, but I'm not sure that's a good use of time right now.